### DISCUSSION TITLE
Shelly EM - duplicato ...

### DISCUSSION THREAD

**Discussion by RobyBS** [indomus/forum#1429] [HUB: Home Assistant]
Buongiorno a tutti ! ... 

avevo uno shelly EM, adibito al controllo del consumo della mia abitazione che dopo un mese di utilizzo circa non ne ha più voluto sapere di funzionare ... 
al che ne ho preso un altro ... 

installato e tutto funzionante ... 
non riesco a "disinstallare" il vecchio shelly EM , o meglio non so come si fa ... ne da APP ne tantomeno da HASSIO ...
nei log mi appare questo messaggio di errore ( che volevo sistemare ) :  
 
```
Logger: homeassistant.components.button
Source: helpers/entity_platform.py:592
Integration: Pulsante (documentation, issues)
First occurred: 31 gennaio 2022, 22:59:00 (2 occurrences)
Last logged: 31 gennaio 2022, 22:59:00

Platform shelly does not generate unique IDs. ID shellyem_c45bbe793438_ota_update already exists - ignoring button.shelly_em_abitazione_ota_update
Platform shelly does not generate unique IDs. ID shellyem_c45bbe793438_reboot already exists - ignoring button.shelly_em_abitazione_reboot

```

oltre che disabilitare le entità precedenti , ma che non risolvono il "conflitto" ... non conosco altre soluzioni .. 

:(

**Comment by snakuzzo** [comment]
https://github.com/StyraHem/ShellyForHASS/issues/307#issuecomment-640082517
